runtime/libia2/include/permissive_mode.h

```diff
@@ -514,11 +518,20 @@ void log_memory_map(void) {
       fprintf(log, "%s", path);
     } else {
       // No path, try to identify it.
+      if (start_addr == ia2_tls_addr_compartment1_first) {
+        fprintf(log, "[tls:tid ?:compartment 1, first]");
+      } else if (start_addr == ia2_tls_addr_compartment1_second) {
+        fprintf(log, "[tls:tid ?:compartment 1, second]");
```

Comment:
I think it's fine to leave out the `first`/`second` from the labels and just have both parts the same.